packages/components/src/modal/Modal.js

```diff
@@ -1,34 +1,54 @@
 // @flow
 
 import * as React from 'react';
-import { View } from 'react-native';
-import { StyleSheet } from '@kiwicom/universal-components';
+import { View, Dimensions, Platform } from 'react-native';
+import { StyleSheet, Modal as UCModal } from '@kiwicom/universal-components';
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 
-import type { ModalProps } from './ModalTypes';
-import ModalWrap from './ModalWrap';
+export type Props = {|
+  +children: React.Node,
+  +isVisible: boolean,
+  +onClose?: () => void,
+|};
 
-const Modal = (props: ModalProps) => {
+export default function Modal({ children, isVisible, onClose }: Props) {
+  const { width, height } = Dimensions.get('window');
+  // Center modal content on web and tablets
+  const centerContent = Platform.OS === 'web' || height / width < 1.6;
   return (
-    <>
-      {props.visible && <View style={styles.overlay} />}
-      <ModalWrap {...props} />
-    </>
+    <UCModal
+      isVisible={isVisible}
+      onRequestClose={onClose}
+      onBackdropPress={onClose}
+      style={[styles.modal, centerContent && styles.modalCentered]}
+    >
+      <View style={[styles.content, centerContent && styles.contentCentered]}>
```

Comment:
This might be problematic if we would like the content to be scrollable. I think it would be worth it to experiment with a ScrollView instead.

Reply:
True, and you could combine it with `alwaysBounceVertical=false` so that it does not scroll unless it should

Reply:
@RobinCsl I've played with your idea for a bit. But at the end I don't think we currently have any use case where scrollable modal will be necessary.

Maybe you've got any specific use-case in mind, just let me know.

The only scrollable content that I've found is place picker and itinerary detail. Where scrolling is solved inside component itself. 

Because place picker have static input on top. With scrollable list underneath.
<img width="393" alt="screenshot 2019-02-19 at 14 58 23" src="https://user-images.githubusercontent.com/2660330/53027062-796be980-3464-11e9-9b47-fd59fc0a6f17.png">

And there is just scrollable list with static button on top in itinerary detail. But whole modal content doesn't need to be scrollable.
![screenshot_20190219-145731](https://user-images.githubusercontent.com/2660330/53027039-6e18be00-3464-11e9-90d7-c59034d5dc0a.png)

This extended modal simply just provides content wrapper with shared style and logic (centered on web/tablet, at the bottom on mobile). 
But while considering all possible usages I've added `wrapperStyle` prop which enables extending wrapper style. So for example wrapper height can be set to fixed value or better to percents, like currently used `70%` for place picker.